### PR TITLE
Avoid AV if schema change doesn't exist.

### PR DESCRIPTION
**Pull request by jkotalik**
Fix for #171
@natemcmaster this should fix the AV if the schema isn't there.

### CODE REVIEW

**Comment by natemcmaster** [comment]
👍

File: src/AspNetCore/Src/aspnetcoreconfig.cxx [hunk 1]
```diff
@@ -241,10 +241,9 @@ ASPNETCORE_CONFIG::Populate(
         &strHostingModel);
     if (FAILED(hr))
     {
-        goto Finished;
+        m_fIsOutOfProcess = TRUE;
```
Comment (pan-wang) [inline src/AspNetCore/Src/aspnetcoreconfig.cxx]: you only need to reset the hr to S_OK

File: src/AspNetCore/Src/aspnetcoreconfig.cxx [hunk 2]
```diff
@@ -241,10 +241,9 @@ ASPNETCORE_CONFIG::Populate(
...
     }
-
-    if (strHostingModel.IsEmpty() || strHostingModel.Equals(L"outofprocess", TRUE))
+    else if (strHostingModel.IsEmpty() || strHostingModel.Equals(L"outofprocess", TRUE))
```
Comment (pan-wang) [inline src/AspNetCore/Src/aspnetcoreconfig.cxx]: no need for this change

File: src/AspNetCore/Src/proxymodule.cxx [hunk 3]
```diff
@@ -85,7 +85,12 @@ CProxyModule::OnExecuteRequestHandler(
     ASPNETCORE_APPLICATION* pAspNetCoreApplication;
     ASPNETCORE_CONFIG::GetConfig(pHttpContext, &config);
 
-    // TODO store whether we are inproc or outofproc so we don't need to check the config everytime?
+    if (config == NULL)
```
Comment (pan-wang) [inline src/AspNetCore/Src/proxymodule.cxx]: no need for this one

**Comment by jkotalik** [comment]
@pan-wang 🆙  📅

**Review by pan-wang — APPROVED** [review]

**Pull request merged**